src/app/page.tsx

```diff
@@ -1,3 +1,10 @@
+import FormWrapper from '@/components/FormWrapper/FormWrapper';
+
 export default function Home() {
-  return <h1>Home page</h1>;
+  return (
+    <>
+      <h1>Home page</h1>
+      <FormWrapper />
```

Comment:
We should probably move the FormWrapper component to the [id] page at `uw-211/src/app/[id]/page.tsx` since that's where the form will be located